ouroboros-consensus-test/bench-mempool/Bench/Consensus/Mempool/Options.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase                 #-}
+{-# LANGUAGE NumericUnderscores         #-}
+
+module Bench.Consensus.Mempool.Options (
```

Comment:
Are we going to be modifying these options on the command line? If most of the options are going to be set in the benchmark themselves perhaps we don't need this.

Reply:
Removing these custom options is straightforward -- I'll let you decide!

Reply:
Right, but the question is if we're ever going to need this :) (YAGNI principle). Since you've already gone through the trouble of writing this module, and since this module only depends on one consensus module I guess we could leave it. The only issue I see is that the default values are specified in this module and not in `Main`, which makes understanding the code a bit harder than it ought to be. I wonder if we could somehow specify the values for `Ns` and `BSS` in the `Main` module instead.

Reply:
I've chosen to remove it! You had me at YAGNI principle